packages/server/src/fhir/operations/patientsummary.ts

```diff
@@ -342,17 +324,17 @@ export class PatientSummaryBuilder {
       custodian: this.patient.managingOrganization,
       event: this.buildEvent(),
       section: [
-        createSection(LOINC_ALLERGIES_SECTION, 'Allergies', this.allergies),
-        createSection(LOINC_IMMUNIZATIONS_SECTION, 'Immunizations', this.immunizations),
-        createSection(LOINC_MEDICATIONS_SECTION, 'Medications', this.medications),
-        createSection(LOINC_PROBLEMS_SECTION, 'Problem List', this.problemList),
-        createSection(LOINC_RESULTS_SECTION, 'Results', this.results),
-        createSection(LOINC_SOCIAL_HISTORY_SECTION, 'Social History', this.socialHistory),
-        createSection(LOINC_VITAL_SIGNS_SECTION, 'Vital Signs', this.vitalSigns),
-        createSection(LOINC_PROCEDURES_SECTION, 'Procedures', this.procedures),
-        createSection(LOINC_DEVICES_SECTION, 'Devices', this.devices),
-        createSection(LOINC_ASSESSMENTS_SECTION, 'Assessments', this.assessments),
-        createSection(LOINC_PLAN_OF_TREATMENT_SECTION, 'Plan of Treatment', this.planOfTreatment),
+        this.createAllergiesSection(),
```

Comment:
Each section now has its own logic for building the "summary" table in the narrative section.